schema/list.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The downloader script is atm just pulling `outpack` schemas; it could pull schemas from the R `outpack.server`, but it wasn't clear to me that that's the canonical place for the api schemas to live as opposed to this repo?

Reply:
absolutely. Perhaps we need an schemas repo to be the single source of truth for everything?